middleware/http/nethttpadaptor/nethttpadaptor.go

```diff
@@ -29,11 +29,11 @@ func NewNetHTTPHandlerFunc(logger logger.Logger, h fasthttp.RequestHandler) http
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		c := fasthttp.RequestCtx{}
 		remoteIP := net.ParseIP(r.RemoteAddr)
-		remoteAddr := net.IPAddr{remoteIP, ""} //nolint
+		remoteAddr := net.IPAddr{remoteIP, ""} // nolint
```

Comment:
Please keep the space here. Comments meant for machines need not to have spaces after `//`

Reply:
Just use the suggestion mode and then you can directly commit the suggestion from the UI here @ItalyPaleAle

Reply:
```suggestion
		remoteAddr := net.IPAddr{remoteIP, ""} //nolint
```

Reply:
I don't have that option :( Probably because I'm not a maintainer